Add "Experimental" to alpha service descriptions

adborden opened the pull request:
https://favro.com/card/1e11108a2da81e3bd7153a7a/18F-3602

adborden committed:
Add "experimental" to alpha service descriptions

adborden committed:
Document where you can find the source metadata

brittag commented on data/services/aws-rds.toml:
```diff
@@ -1,3 +1,5 @@
+# Service metadata lives
+# https://github.com/18F/aws-broker/blob/59f949b/catalog-template.yml#L8
```
Nice idea! Suggest using the branch URLs - such as https://github.com/18F/aws-broker/blob/master/catalog-template.yml#L8

adborden replied:
I usually avoid branch URLs because then the line number can change or file can be moved and then the user is totally in the dark after clicking the link. Users can track a commit through it's history to find where the line moved to if they're looking for recent history.

brittag replied:
Interesting point! I think in general if the file moves, we should update the links in these comments - my hope would be that the editor of this file would be able to hop over to the referenced file and edit the description to match.

brittag replied:
Actually the best resolution for this might be to skip thinking about the "best" strategy, keep these URLs since they definitely serve as a helpful hint, and hope for the glorious future of https://favro.com/card/1e11108a2da81e3bd7153a7a/18F-3375 :)

adborden committed:
rfc

Remove uaa-auth toml data, the service itself is no longer supported and docs
have already been removed, this file was just missed.
